lib/net/peer.js

```diff
@@ -1960,6 +1971,13 @@ class Peer extends EventEmitter {
       'Sending proof (%s).',
       this.fullname());
 
+    this.proofWindow.increase(1);
+    if (!this.proofWindow.allow()) {
+      this.logger.debug('proof: rate limit exceeded (%s).', this.hostname());
```

Comment:
This logger statement should use `realname` and a call to `fullname` is still in the code above